src/utils.js

```diff
@@ -7,18 +7,33 @@ function getTimeString(unix_time) {
     const hours = ("0" + date.getHours()).slice(-2);
     const minutes = ("0" + date.getMinutes()).slice(-2);
     const seconds = ("0" + date.getSeconds()).slice(-2);
-    return hours + ":" + minutes + ":" + seconds;
+
+    return hours + ":" + minutes + ":" + seconds; //because timepicker likes it like this
 }
 
 function getDateString(unix_time) {
     const date = unixToDate(unix_time);
     const day   = ("0" + date.getDate()).slice(-2);
     const month = ("0" + (date.getMonth() + 1)).slice(-2);
     const year  = date.getFullYear();
-    return year + "-" + month + "-" + day;
+
+    return year + "-" + month + "-" + day; //because datepicker likes it like this
+}
+
+function dateTimeToUnix(date, time) {
```

Comment:
@vritant24 Wasn't aware of this. Should I use this as opposed to `.toLocaleString()`?